<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_i_inventory_check_Dialog</class>
 <widget class="QDialog" name="w_i_inventory_check_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color:purple;</string>
        </property>
        <property name="text">
         <string>开始时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_datetimeEdit" name="dateTimeEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color:purple;</string>
        </property>
        <property name="text">
         <string>结束时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_datetimeEdit" name="dateTimeEdit_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">color:purple;</string>
        </property>
        <property name="text">
         <string>商品类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_ComboBox" name="comboBox_series">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">color:purple;</string>
        </property>
        <property name="text">
         <string>商品编码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="lds_selectgoods" name="selectgoods" native="true"/>
      </item>
      <item>
       <widget class="lds_toolbar" name="widget"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string/>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="lds_tableview_sum" name="tableView"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">color:purple;</string>
         </property>
         <property name="text">
          <string>制表人</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_operator">
         <property name="text">
          <string>制表人</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">color:purple;</string>
         </property>
         <property name="text">
          <string>制表时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_operattime">
         <property name="text">
          <string>制表时间</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar</class>
   <extends>QFrame</extends>
   <header>lds_toolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lds_tableview_sum</class>
   <extends>QTableView</extends>
   <header>lds_tableview_sum.h</header>
  </customwidget>
  <customwidget>
   <class>lds_datetimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>lds_datetimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_selectgoods</class>
   <extends>QWidget</extends>
   <header>lds_selectgoods.h</header>
   <container>1</container>
   <slots>
    <signal>okstr1(QString)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>dateTimeEdit_2</tabstop>
  <tabstop>comboBox_series</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>dateTimeEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
